environments/demo/demo.tfvars

```diff
@@ -3,7 +3,12 @@ subscription           = "demo"
 cft_apps_cluster_ips   = ["10.50.79.221", "10.50.95.221"]
 certificate_name_check = false
 autoShutdown           = true
-ssl_certificate        = "wildcard-demo-platform-hmcts-net"
+ssl_policy = {
```

Comment:
not matching variable name used, as mentioned yesterday - this will attempt to update on backend appgw